passky-client/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3.7"
+
+services:
+
+  app_proxy:
+    environment:
+      APP_HOST: $APP_PASSKY_CLIENT_IP
+      APP_PORT: 80
+      PROXY_AUTH_ADD: "false"
```

Comment:
Hello,

Protecting your application with Umbrel app proxy is great, but for applications that already have login system build in, it would be kinda unnecessary as users will then need to enter 2 separated passwords and 2 separated 2FA (If they are enabled it in Umbrel and also in Passky Server).

Passky Client is also meant to be distributed to everyone publicly as it does not contain / store any user data and this also won't impact user security.

I need to apologize in advance as my English is really bad, but I hope you understands me.

Basically even if you deploy your Passky Client publically as this instance on Cloudflare as an example: https://vault.passky.org it won't impact any security as JS will process all data on your computer / client side. The only problem that can happen is if someone manipulate with this JS code.

So hosting Passky Client is completelly safe for everyone that host it on their own. But users that will use your hosted Passky Client, would need to trust you or check for themselves if your Passky Client does not contain any modifications. But if you already come to this point, it will just be better for each user to just use Passky Client in form of desktop application, mobile application or browser extension). So html, css and JS files will be kept offline and won't be pulled from a server every once in a while when cache will expire.